deploy/kicbase/Dockerfile

```diff
@@ -124,6 +124,7 @@ RUN sh -c "echo 'deb https://download.docker.com/linux/ubuntu focal stable' > /e
 # install buildkit
 RUN export ARCH=$(dpkg --print-architecture | sed 's/ppc64el/ppc64le/' | sed 's/armhf/arm-v7/') \
  && echo "Installing buildkit ..." \
+    && addgroup --system buildkit \
     && export BUILDKIT_BASE_URL="https://github.com/moby/buildkit/releases/download/${BUILDKIT_VERSION}" \
     && curl -sSL --retry 5 --output /tmp/buildkit.tgz "${BUILDKIT_BASE_URL}/buildkit-${BUILDKIT_VERSION}.linux-${ARCH}.tar.gz" \
```

Comment:
Almost forgot, you also have to download and install the systemd unit files: (to /usr/local/lib/systemd/system/)
https://raw.githubusercontent.com/moby/buildkit/v0.9.0/examples/systemd/buildkit.service
https://raw.githubusercontent.com/moby/buildkit/v0.9.0/examples/systemd/buildkit.socket
Too bad there is no decent packaging (like a .deb) for buildkit... (See #10068)

Reply:
We want to to the minikube changes as overrides, so that we can leave the upstream files intact (see podman).

https://wiki.archlinux.org/title/Systemd#Drop-in_files

Reply:
Eventually the /etc/buildkit will have to be populated here as well

Reply:
Thanks for the guidance @afbjorklund! I implemented downloading and installing the systemd unit files and added the `/etc/buildkit` for kicbase as well, I tested the changes and successfully built the image using `image build`. I created the `/etc/buildkit` file by echoing to a file, I'm not sure if there was somewhere specific you want to put the file and copy it over so please take a look at my changes and let me know.

But the gist is `image build` via kicbase was successful, onto iso.